web/extensions/core/widgetInputs.js

```diff
@@ -228,7 +228,7 @@ app.registerExtension({
 					const linkInfo = app.graph.links[l];
 					const node = this.graph.getNodeById(linkInfo.target_id);
 					const input = node.inputs[linkInfo.target_slot];
-					const widgetName = input.widget.name;
+					const widgetName = input.widget?.name;
```

Comment:
This fixes a bug in the front-end when an input that would generally have an associated widget is explicitly declared as having a socket.